### PR TITLE
PR Template: Fix typos, rephrase things, eliminate redundant section

### PR DESCRIPTION
**Pull request by eecavanna**
Hi @mslarae13, here are some changes I propose to the PR template in https://github.com/microbiomedata/berkeley-schema-fy24/pull/217. I created this PR instead of directly committing these changes onto the branch you're working on. If you want to adopt these changes, you can merge this PR branch _into that branch._

### CODE REVIEW

**Comment by eecavanna** [comment]
You can view the "rendered" version of the PR template, here:

https://github.com/microbiomedata/berkeley-schema-fy24/blob/eecavanna-patch-1/.github/PULL_REQUEST_TEMPLATE/pull_request_template_berk.md

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template_berk.md [hunk 1]
```diff
@@ -1,86 +1,82 @@
 # Guidelines
 
 ## _Soft_ Schema Freeze
-The nmdc-schema and berkeley-schema are under a light freeze this means changes **should not** be made that have any down stream implications. To ensure this, PRs need to be closely reviewed with **every** component of the NMDC in mind.
+
+The `nmdc-schema` and `berkeley-schema-fy24` schemas are under a light freeze, which means changes **should not** be made that have any downstream implications. To ensure this, all PRs created creating during the freeze will be closely reviewed with **every** component of the NMDC system in mind.
```
Comment (mslarae13) [inline .github/PULL_REQUEST_TEMPLATE/pull_request_template_berk.md:5]: I missed this one... light -> soft

**Review by mslarae13 — APPROVED** [review]
One small update I missed, but also I can just make that change on my side

**Comment by mslarae13** [comment]
Actually, I want to wrap this up. I'm going to merge and I'll make the change from light to soft in my branch

**Pull request merged**